subworkflows/nf-core/vcf_annotate_snpeff/main.nf

```diff
@@ -0,0 +1,28 @@
+//
+// Run SNPEFF to annotate VCF files
+//
+
+include { SNPEFF           } from '../../../modules/nf-core/snpeff/main.nf'
+include { TABIX_BGZIPTABIX } from '../../../modules/nf-core/tabix/bgziptabix/main.nf'
+
+workflow VCF_ANNOTATE_SNPEFF {
+    take:
+    vcf          // channel: [ val(meta), vcf ]
+    snpeff_db    //   value: db version to use
+    snpeff_cache //    path: /path/to/snpeff/cache (optionnal)
```

Comment:
`[meta2, snpeff_cache]`?

Reply:
no need for cache, it's the root path to folder

Reply:
cache is then resolved for each genome depending of the snpeff_db value